src/store/ducks/opensrp/planIdsByUser/index.ts

```diff
@@ -0,0 +1,157 @@
+import { Dictionary } from '@onaio/utils';
+import { AnyAction, Store } from 'redux';
+import { createSelector } from 'reselect';
+import SeamlessImmutable from 'seamless-immutable';
+import { PlanDefinition } from '../../../../configs/settings';
+
+/** the reducer name */
+export const reducerName = 'planIdsByUser';
```

Comment:
What's preventing us from using the already existing plans reducer module?

Reply:
the plans reducer module stores plans with the `PlanRecord` type, but the plans that we get from the filter endpoint are of `PlanDefinitionList` type and are handled by the opensrp/PlanDefinition reducer.

Reply:
Cool thanks, does that mean that we can use this reducer https://github.com/onaio/reveal-frontend/blob/89e8ef36c9a6f2fa036d598630bb6082ad3bfc56/src/store/ducks/opensrp/PlanDefinition/index.ts